src/sagas/messages/sendMessage/sendMessage.saga.ts

```diff
@@ -32,8 +26,10 @@ export function* sendMessageSaga(
     ReturnType<typeof messagesActions.sendMessage>['payload']
   >
 ): Generator {
-  const identity = yield* select(identitySelectors.currentIdentity);
-  if (!identity.userCertificate) {
+  const identity: Identity = yield* select(identitySelectors.currentIdentity);
+  
+  const certificate = identity.userCertificate;
+  if (!certificate) {
```

Comment:
I think we don't need this condition anymore

Reply:
Yeah... user will never get to that place without identity